tests/modules/programs/thunderbird/thunderbird.nix

```diff
@@ -1,6 +1,8 @@
-{ pkgs, ... }: {
+{ lib, realPkgs, ... }: {
   imports = [ ../../accounts/email-test-accounts.nix ];
 
+  _module.args.pkgs = lib.mkForce realPkgs;
+
```

Comment:
This is removing the refactored testing stubbing, we should just be inheriting from realPkgs where needed.